### ISSUE TITLE
Oxford Comma?

### ISSUE THREAD

**Issue by ericclemmons** [component/join#1]
Shouldn't the join use a clarifying ([Oxford](http://oxforddictionaries.com/words/what-is-the-oxford-comma)) comma?

``` js
join(['foo', 'bar', 'raz'], 'or').should.equal('foo, bar, or raz')
```

**Comment by ericclemmons** [comment]
Looking at the code, then perhaps only an example is needed:

``` js
join(['foo', 'bar', 'raz'], ', or').should.equal('foo, bar, or raz')
//                           ^-- check out that comma!
```

**Comment by tj** [comment]
doesn't matter to me, my english is poor haha, whichever is more "correct" in this bastard of a language

**Comment by TooTallNate** [comment]
+1 for a simple example in the README. This should be opt-in IMO.

**Comment by ericclemmons** [comment]
Since it's personal preference, I'll add an example to the README.  It _is_ personal preference, though working with a huge team of writers I've learned to love that additional comma.

Look at us Javascripters!  Still arguing over where to place commas! :D

**Comment by ericclemmons** [comment]
Closed via #2
